### PR TITLE
spectra_ref column for PEP section in mzTab (MzTabExporter)

### PR DESCRIPTION
**Pull request by lars20070**
When exporting `consensusXML` to `mzTab` with `MzTabExporter`, the `spectra_ref` column (containing `mzML` file name and spectral index) was missing. This PR fills in this information.

### CODE REVIEW

File: src/openms/include/OpenMS/FORMAT/MzTab.h [hunk 1]
```diff
@@ -944,7 +944,9 @@ namespace OpenMS
         const std::vector<ProteinIdentification>& prot_ids,
```
Comment (hroest) [inline src/openms/include/OpenMS/FORMAT/MzTab.h]: please add docu that explains the different parameters, especially what the maps do

**Comment by hroest** [comment]
please merge into `develop` and update with documentation

File: src/openms/include/OpenMS/FORMAT/MzTab.h (hunk 1 above)
Comment (lars20070) [reply src/openms/include/OpenMS/FORMAT/MzTab.h]: Just FYI, the helper maps are not new. But now just exposed.
https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/FORMAT/MzTab.cpp#L1775
I will add further docu.

**Comment by lars20070** [comment]
@timosachsenberg Can you please check the extended docu?

**Comment by lars20070** [comment]
For clarification, only identified consensus features report `spectra_ref` in the `PEP` section, i.e. the file and spectrum index corresponding to that PSM. Consensus features without ID report `null` in the `spectra_ref` column.

**Pull request merged**